contracts/actions/curve/helpers/CurveHelper.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "../../../interfaces/curve/IAddressProvider.sol";
+import "../../../interfaces/curve/ISwaps.sol";
+import "../../../interfaces/curve/IMinter.sol";
+import "../../../interfaces/curve/IVotingEscrow.sol";
+
+contract CurveHelper {
+    address public constant CrvTokenAddr = 0xD533a949740bb3306d119CC777fa900bA034cd52;
```

Comment:
Use capital notation with underscore for constant addresses. For instance `CRV_TOKEN_ADDR`

Reply:
DONE